File: Modul 7/SOAL 3/Bahasa C/PRAK603-2210817210028-Muhammad Aulia Rasyid.c
#include <stdio.h>
int main (void)
{
    int baris1, baris2, a, isi ;
    scanf("%d %d", &baris1, &baris2) ;
    int angka1[baris1], angka2[baris2] ;
    if (baris1 != baris2) {
        printf ("\nJumlah tidak sama") ;
    }
    else {
        for (a = 0; a < baris1; a++) {
            scanf ("%d", &isi) ;
            angka1[a] = isi ;
        }
        for (a = 0; a < baris2; a++) {
            scanf ("%d", &isi) ;
            angka2[a] = isi ;
        }
        printf ("\n") ;
        for (a = 0; a < baris1; a++) {
            printf ("%d ", angka1[a] * angka2[a]) ;
        }
    }
}

File: Modul 7/SOAL 3/Bahasa Python/PRAK603-2210817210028-Muhammad Aulia Rasyid.py
baris1, baris2 = map(int, input().split())
if (baris1 != baris2) :
    print("\nJumlah tidak sama")
else :
    isi1 = list(map(int, input().split()))
    isi2 = list(map(int, input().split()))
    angka1 = [] ; angka2 = []
    print()
    for a in range(0, baris1) :
        angka1.append(isi1[a])
    for a in range(0, baris2) :
        angka2.append(isi2[a])
    for a in range(0, baris1) :
        print(angka1[a] * angka2[a], end=' ')